src/libtools/property.cc

```diff
@@ -466,6 +469,7 @@ std::ostream &operator<<(std::ostream &out, const Property &p) {
     switch (type) {
       default:
         PrintNodeTXT(out, p, level);
+        break;
```

Comment:
@JensWehner hopefully we never used txt output anywhere....

Reply:
do not think so.